webapp/actions/websocket_actions.jsx

```diff
@@ -241,6 +245,12 @@ function handleUserRemovedEvent(msg) {
     }
 }
 
+function handleUserUpdatedEvent(msg) {
+    if (UserStore.getCurrentId() !== msg.user_id) {
+        AsyncClient.getProfiles();
```

Comment:
Worried about scalability on this one. Do we really want to pull all profiles when any user is updated?